### ISSUE TITLE
Something isn't instanced within FeatureVillageStargate.java

### ISSUE THREAD

**Issue by Dockter** [dallenwilson/SGCraft#2]
Got this error:

https://gist.github.com/Dockter/becec51f802d3719c9a44734b38ca8a1

**Comment by Dockter** [comment]
Also, the { breaks are in the wrong place based on the coding style of the rest of the project.

**Comment by dallenwilson** [comment]
Force of habit, I'm used to GNU-style C. Easily corrected, thanks for pointing it out.

I don't get any errors no matter how many villages I visit, but your build environment seems to be quite a bit different than the bare-bones one I have set up. I'll dig in to the class and see if I missed something, I looked at ImmersiveEngineering's code for the basic how-to as far as village structures went.

**Comment by dallenwilson** [comment]
I started with a fresh clone of your repo and almost have a working build system. Compiling and launching from within eclipse using the SGCraft_Client run configuration gives me a similar tick exception error, but from NaquadahOreWorldGen. Using my original build process (A shell script that runs ./gradlew --offline build, copies the complied mod to my usual ~/.minecraft/mods/1.12.2 folder and starts the minecraft-launcher), your mod builds and runs without error.

I don't use or plan to use eclipse, but it would be nice to have it working. I'll keep poking at it. Once I can get your original repo building as intended, I'll start porting my changes over and go from there. In the meantime I closed my pull request.
